book/content/software_checklist/software_checklist.md

```diff
@@ -26,7 +26,7 @@ Additional steps depend on the goal of the software (zero or more can apply):
 ## I'm publishing a paper
 
 * [Make your software citable](/software_checklist/01/making_software_citable)
-* [???] Add a [CITATION.cff file](documentation.html#software-citation)
+* Add a CITATION.cff file (see the "[Further reading](#further-reading)" section at the end of the chapter)
```

Comment:
Minor suggestion: mention `cff` files in third place

Registering a DOI in Zenodo is easier and, in my opinion, more important than having a `cff` file.

Reply:
Fixed in commit 0d3c8ee80dbb576866f1498899b0e82f2957aa6a